### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by kkmanuu**
✔️ I Create a new app. Give it a meaningful name.
▶️ I Initialize your project with Git.
▶️ I Make sure that your project has Postgres database set up.
▶️ I Build MY  project schema.
▶️ I Create and run the necessary migration files.
▶️ Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
✔️:Foreign keys should be included.
✔️attention that in posts table the column holding the foreign key to the users table should be named author_id
All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20231003124327_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
```
Comment (cynthiainga) [inline db/migrate/20231003124327_create_comments.rb]: - [x] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) this is supposed to create `user_id` and a corresponding index, for you it creates an `author_id`. Kindly fix that 🙃 🙃

File: db/migrate/20231003124530_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
```
Comment (cynthiainga) [inline db/migrate/20231003124530_create_likes.rb]: - [x] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) this is supposed to create user_id and a corresponding index, for you it creates an author_id. Kindly fix that 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @kkmanuu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Database adapter is using Postgresql ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20231003124327_create_comments.rb (hunk 1 above)
Comment (kkmanuu) [reply db/migrate/20231003124327_create_comments.rb]: @cynthiainga Done making the changes

File: db/migrate/20231003124530_create_likes.rb (hunk 2 above)
Comment (kkmanuu) [reply db/migrate/20231003124530_create_likes.rb]: @cynthiainga Done making the changes

**Review by cynthiainga — APPROVED** [review]
Hi @kkmanuu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] All changes are made ✔️ 
- [ ] No linters errors ✔️
- [ ] Correct Gitflow ✔️
- [ ] Database adapter is using Postgresql ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**